invenio_rdm_records/services/pids/providers/base.py

```diff
@@ -8,13 +8,15 @@
 
 """PID Provider."""
 
+from abc import ABC, abstractmethod
+
 from flask import current_app
 from flask_babelex import lazy_gettext as _
 from invenio_pidstore.errors import PIDAlreadyExists, PIDDoesNotExistError
 from invenio_pidstore.models import PersistentIdentifier, PIDStatus
 
 
-class PIDProvider:
+class PIDProvider(ABC):
```

Comment:
This was *very* handy to spot the changes needed and will immediately raise the need to implement that method for any third-party provider implementers.